File: CabBookingApplication/src/main/java/com/cg/mts/dto/DriverDTO.java
package com.cg.mts.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class DriverDTO {
	
	int driverId;
	String driverName;
	String licenceNo;
	float rating;
	String nativeLanguage;
	int experience ;

}

File: src/app/driver-dto.ts
export class DriverDto {

    driverName:string;
    licenceNo:string;
    rating:number;
    category:string;
    city:string;
    experience:number;
    nativeLanguage:string;


     constructor(
         driverName:string,
         licenceNo:string,
         rating:number,
         category:string,
         city:string,
         experience:number,
         nativeLanguage:string,

     )
     {
         this.driverName = driverName;
         this.licenceNo = licenceNo;
         this.rating = rating;  
         this.category = category;
         this.city = city;
         this.experience = experience;
         this.nativeLanguage = nativeLanguage;

     }
}
